core-web/libs/portlets/dot-experiments/portlet/src/lib/dot-experiments-reports/components/dot-experiments-reports-chart/chartjs/plugins/dotHtmlLegend-chartjs.plugin.ts

```diff
@@ -1,7 +1,11 @@
 const COLOR_PALETTE_GRAY_100 = '#fafafb';
```

Comment:
Maybe we can get this color using JS with this [example ](https://stackoverflow.com/questions/41725725/access-css-variable-from-javascript)

Reply:
Super, used.